redbeams/src/main/java/com/sequenceiq/redbeams/flow/redbeams/start/actions/StartDatabaseServerFinishedAction.java

```diff
@@ -28,11 +28,11 @@ public class StartDatabaseServerFinishedAction extends AbstractRedbeamsStartActi
     private RedbeamsMetricService metricService;
 
     public StartDatabaseServerFinishedAction() {
-        super(StartDatabaseServerSuccess.class);
+        super(CertRotateInRedbeamsSuccess.class);
     }
 
     @Override
-    protected void prepareExecution(StartDatabaseServerSuccess payload, Map<Object, Object> variables) {
```

Comment:
is this backward compatible? The payload changed for the state.

Reply:
during the restart, the StartDatabaseServerSuccess event will go inside the CertRotateInRedbeamsAction instead of StartDatabaseServerFinishedAction. so it will work properly. 
Yes, I know It cannot be tested or is very hard to test, but the logic should work and we also add steps into the flows

Reply:
If we are at REDBEAMS_START_FINISHED_STATE and we restart it, then the flow engine will try to deserialize CertRotateInRedbeamsSuccess, but the payload content is serialized from StartDatabaseServerSuccess. Isn't it?

Reply:
I have tested and it works as expected